File: lua/1015.lua
local math = require("math")

function calcularDistancia(x1, y1, x2, y2)
    return math.sqrt((x2 - x1)^2 + (y2 - y1)^2)
end

local x1, y1, x2, y2
x1 = io.read("*n")
y1 = io.read("*n")
x2 = io.read("*n")
y2 = io.read("*n")

local distancia = calcularDistancia(x1, y1, x2, y2)
print(string.format("%.4f", distancia))

File: go/1015.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1, x2, y1, y2, d float64
	fmt.Scanf("%f", &x1)
	fmt.Scanf("%f", &y1)
	fmt.Scanf("%f", &x2)
	fmt.Scanf("%f", &y2)
	d = math.Sqrt(math.Pow((x2-x1), 2) + math.Pow((y2-y1), 2))
	fmt.Printf("%.4f\n", d)
}
